### ISSUE TITLE
Support for PocketBeagle on 4.4

### ISSUE THREAD

**Issue by giuliomoro** [RobertCNelson/Bootloader-Builder#12]
Hello,
I cannot see any support for the PocketBeagle on 4.4 (specifically, we are interested in the `capemgr`). Is this just not going to happen? Any pointers as to where to start to backport it?

**Comment by RobertCNelson** [comment]
Um, confused:

https://github.com/beagleboard/linux/blob/4.4/arch/arm/boot/dts/am335x-pocketbeagle.dts

Otherwise, "capemgr" use u-boot overlays on the PocketBeagle..

Regards,

**Comment by giuliomoro** [comment]
I guess "any" was the wrong choice of word above.
Let me rephrase: I cannot see any capmgr+PB-related patches in your 4.4.y branch, while they seem be there on 4.9 and 4.14. I guess we'll just have to go for uboot overlays?

**Comment by RobertCNelson** [comment]
There's no capemgr+PB patches in 4.9.x or 4.14.x.. ;)

**Comment by giuliomoro** [comment]
ok brilliant. And sorry for opening it in the wrong repo.
